### PR TITLE
[BPK-4310] Update calendar snapshot test size

### PR DESCRIPTION
**Pull request by georgegillams**

### CODE REVIEW

File: Example/SnapshotTests/BPKCalendarSnapshotTest.m [hunk 1]
```diff
@@ -60,11 +61,11 @@ - (void)configureParentView:(UIView *)parentView forCalendar:(BPKCalendar *)cale
                                                                        options:0
                                                                        metrics:nil
                                                                          views:@{@"calendar": calendar}]];
-    [parentView addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"[parentView(320)]"
```
Comment (k0nserv) [inline Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: Does this mean the current calendar design will not work on the previous generation of iPhone SE?

Comment (georgegillams) [reply Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: You mean for price labels?
I guess if the iPhone SE has a screen width of 320 then no 🤷

Comment (k0nserv) [reply Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: Okay that's probably worth highlighting too

**Review by k0nserv — APPROVED** [review]
LGTM

File: Example/SnapshotTests/BPKCalendarSnapshotTest.m (hunk 1 above)
Comment (georgegillams) [reply Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: Do we still support iOS 12?

Comment (georgegillams) [reply Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: iOS 13 supports the SE 😮

Comment (k0nserv) [reply Example/SnapshotTests/BPKCalendarSnapshotTest.m:63]: Some iPod is also 320pt wide I believe

**Pull request merged**